python-flask/python-flask-example.py

```diff
@@ -13,20 +13,18 @@ def log_payload(path):
         return 'Content type is not JSON', 415
 
     # Ensure that payload is a dictionary
-    payload = request.json
-    if not isinstance(payload, dict):
+    data = request.json
+    if not isinstance(data, dict):
         return 'Payload is not JSON', 500
 
     # Get message data
-    timestamp = datetime.datetime.now()
-    message = payload.get('message',{})
-    if message:
+    device_id = data.get('deviceId', {})
+    if device_id:
```

Comment:
Could this be a `try` instead? We should really always be sending a `deviceId` anyway.

Reply:
Sure, updated original commit